src/renderer/lib/hooks/useAppListeners.ts

```diff
@@ -229,7 +229,19 @@ export const useAppListeners = () => {
 
   const setDolphinOpen = useDolphinStore((store) => store.setDolphinOpen);
   ipc_dolphinClosedEvent.renderer!.useEvent(
-    async ({ dolphinType }) => {
+    async ({ dolphinType, exitCode }) => {
+      if (exitCode !== 0) {
+        addToast(
+          `Dolphin encountered an error with code: 0x${exitCode.toString(16)}.
+          If you are having issues, please screenshot this and post it in a support channel in the
+          Slippi Discord.`,
```

Comment:
```suggestion
          `Dolphin exited with error code: 0x${exitCode.toString(16)}. Visit the Slippi Discord for help with this error.`,
```
Thoughts on simplifying the error message copy to just this?

Reply:
It's shorter but less clear. I think people will show up after dismissing the message and not writing down the code.